alluxio-ui/master/src/containers/pages/Metrics/Metrics.tsx

```diff
@@ -299,6 +301,15 @@ export class Metrics extends React.Component<AllProps> {
                 </Table>
               </div>
             ))}
+            <div className="col-12">
+              <div className="row">
+                {data.timeSeriesMetrics.map((metric: LineSerieData) => (
+                  metric.data.length && <LineGraph key={metric.id} data={[metric]} xAxisLabel={metric.xAxisLabel}
+                                                   xAxisUnits={metric.xAxisUnits} yAxisLabel={metric.yAxisLabel}
+                                                   yAxisUnits={metric.yAxisUnits}/>
+                ))}
+              </div>
+            </div>
```

Comment:
The two graphs we currently have can replace the `Master Gauges` section at the top of this page.